tests/test_core.py

```diff
@@ -6457,7 +6457,7 @@ def line_splitter(data):
                               Path('codec/CMakeFiles/j2k_to_image.dir/convert.c.o'),
                               Path('codec/CMakeFiles/j2k_to_image.dir/__/common/color.c.o'),
                               Path('bin/libopenjpeg.a')],
-                             configure=['cmake', '.'],
+                             configure=['cmake', '.', '-DBUILD_SHARED_LIBS=OFF'],
```

Comment:
Why is this needed if the default is set of OFF above?

Reply:
I wonder as well. Without it the `test_openjpeg`  does not seem to build
```
libopenjpeg/CMakeFiles/openjpeg.dir/build.make:388: bin/libopenjpeg.so.1.4.0
```